web/pages/index.vue

```diff
@@ -6,18 +6,21 @@
           <SchedulerConfigurationStoreProvider>
             <StorageClassStoreProvider>
               <PriorityClassStoreProvider>
-                <SnackbarStoreProvider>
-                  <ResourceBar />
-                  <SchedulerConfigurationEditButton />
-                  <ResourceAddButton />
-                  <NodeList />
-                  <UnscheduledPodList />
-                  <PVList />
-                  <PVCList />
-                  <StorageClassList />
-                  <PriorityClassList />
-                  <Snackbar />
-                </SnackbarStoreProvider>
+                <ResetProvider>
+                  <SnackbarStoreProvider>
+                    <ResourceBar />
+                    <SchedulerConfigurationEditButton />
+                    <ResourceAddButton />
+                    <ResetButton />
```

Comment:
For the position of reset button, I prefer to place it next to import/export button.
So, let's wait for #81 and place it on [the topbar](https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/81/files#diff-9af398f53fe84b85281985ad230feb7d7c7bccc7f2dca0a4a96b54e81f59a3a5). I think #81 is about to merge.

Reply:
OK!, I wait it.

Reply:
@sivchari 
We merged #81.